dvc/logger.py

```diff
@@ -146,14 +143,6 @@ def _parse_exc(self, exc_info):
 
         return (exception, stack_trace)
 
-    def _progress_aware(self):
```

Comment:
What if we need to print something (e.g. debug/warning message) while in the progress bar? Would that break the progress bar?

Reply:
technically yes, will need to update based on

- https://github.com/tqdm/tqdm/#writing-messages
- https://github.com/tqdm/tqdm/#redirecting-writing
- https://github.com/tqdm/tqdm/blob/master/examples/redirect_print.py
- https://github.com/tqdm/tqdm/issues/786
- https://github.com/tqdm/tqdm/issues/749